### DISCUSSION TITLE
The GitHub search can't search on repository's README

### DISCUSSION THREAD

**Discussion by Truongphi20** [community/community#59308] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm searching on GitHub for the keywords: "**HPV**", and "**virus**" with nextflow language. And hoping my search result has the repo [nf-VIF](https://github.com/bioinfo-pf-curie/nf-vif/). Cause in nf-VIF repo's README file has these keywords. 

[![enter image description here][1]][1]

However, it returns just another result with keywords mapped with the description part. I also tried searching by Github Search API according to [this answer](https://stackoverflow.com/questions/74363221/how-do-i-search-the-readme-of-repositories-using-github-search-api). But it still not worked (https://api.github.com/search/repositories?q=HPV+virus). 

[![enter image description here][2]][2] 

Could someone tell me how to search repo based on README in GitHub, please? Thanks. 

  [1]: https://i.stack.imgur.com/gqYQd.png
  [2]: https://i.stack.imgur.com/Y4BmN.png

**Comment by look** [comment, marked as answer]
README files are not searched by default for repository searches. Try using `in:readme`: [`HPV virus in:readme language:Nextflow`](https://github.com/search?q=HPV+virus+in%3Areadme+language%3ANextflow+&type=repositories). (I see your repository when doing this search.)
